website/_i18n/en/user-guides/use-wallet-cli/index.md

```diff
@@ -21,114 +22,119 @@ To get started, download and extract the Pactus CLI file from
 In this section, we will explore various wallet commands that help you create, manage,
 and recover your Pactus wallet.
 
+{% alert_start info %}
+  <b>Wallet Path</b>
+  <br><br>
+  By default, the node's wallet is stored at `~/pactus/wallets/default_wallet` on Unix-like systems.
+  On Windows, you can find it at `C:\Users\<USER_NAME>\pactus\wallets\default_wallet`.
+  <br><br>
+  If you wish to select or specify a different wallet, use the `--path` option.
+{% alert_end %}
+
 ### Create a Wallet
 
-First, you need to create a wallet. To create a new wallet, run the following command:
+To create a new wallet, run the following command:
 
 {% tabs create_wallet ltr %}
 
 {% tab create_wallet linux-mac <i class="fa-brands fa-linux"></i> Linux / <i class="fa-brands fa-apple"></i> macOS %}
 
 ```text
-./pactus-wallet --path ~/pactus/wallets/default_wallet create
+./pactus-wallet --path <PATH-To-NEW-WALLET> create
 ```
 
-This command creates a new wallet and stores it in the `~/pactus/wallets/default_wallet`.
-
 {% endtab %}
 
 {% tab create_wallet windows <i class="fa-brands fa-windows"></i> Windows %}
 
 ```text
-pactus-wallet.exe c:\pactus\wallets\default_wallet create --testnet
+pactus-wallet.exe --path <PATH-To-NEW-WALLET> create --testnet
 ```
 
-This command creates a new wallet and stores it in the `c:\pactus\wallets\default_wallet`.
-
 {% endtab %}
 
 {% endtabs %}
 
-Don't forget to back up your wallet seed, as it allows you to recover your wallet if you lose it.
+Replace `<PATH-To-NEW-WALLET>` with the file path where you want to create the wallet.
 
-### Wallet Password
+### Recover Wallet
 
-You can change the wallet password by running this command:
+If you lose your wallet or forget your password, you can recover it by running this command:
```

Comment:
@b00f 
should not be `lose` -> `lost` also `forget` -> `forgot` ?

Reply:
@Mr-HoDL58 which one is better in your opinion? lost or lose?